featuresScore.java

```diff
@@ -0,0 +1,76 @@
+
+public class featuresScore {
+	public static final int COLS = 10;
+	public static final int ROWS = 21;
```

Comment:
I think it is just their implementation detail. For some reason they check for game end by checking `row >= 21` instead of just `row > 20`.... I suggest to just stick to 20 rows